acados/ocp_qp/ocp_qp_common.h

```diff
@@ -69,6 +69,7 @@ typedef struct
     void (*memory_get)(void *config_, void *mem_, const char *field, void* value);
     acados_size_t (*workspace_calculate_size)(void *config, void *dims, void *opts);
     int (*evaluate)(void *config, void *qp_in, void *qp_out, void *opts, void *mem, void *work);
+    void (*solver_get)(void *config_, void *qp_in_, void *qp_out_, void *opts_, void *mem_, const char *field, int stage, void* value);
```

Comment:
I guess this should be initialized also for the QP solvers other than HPIPM, otherwise you would get a segfault or something.

Reply:
Addressed in https://github.com/acados/acados/pull/975/commits/4a8c43cddc7292876b8d5628ad1253f7857703cb